core/src/main/java/org/mule/api/metadata/MetadataNotification.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.api.metadata;
+
+import org.mule.context.notification.CustomNotification;
+
+public class MetadataNotification extends CustomNotification
```

Comment:
you don't need a MEtadataNotification.. you need a ConfigurationInstanceNotification